drivers/spi/spi_nrfx_spim.c

```diff
@@ -35,6 +37,16 @@ LOG_MODULE_REGISTER(spi_nrfx_spim, CONFIG_SPI_LOG_LEVEL);
 #define SPI_BUFFER_IN_RAM 1
 #endif
 
+#if ((DT_NODE_HAS_STATUS(DT_NODELABEL(spi120), okay)) || \
+	(DT_NODE_HAS_STATUS(DT_NODELABEL(spi121), okay))) && defined(HALTIUM_XXAA)
```

Comment:
we should likely avoid using HAL definitions here (HALTIUM_XXAA) and use Zephyr SOC definitions

Reply:
fixed